setup.py

```diff
@@ -7,11 +7,11 @@
 import setuptools
 
 sys.path.insert(0, ".")
-from chaosazure import __version__
 sys.path.remove(".")
 
 name = 'chaostoolkit-azure'
 desc = 'Chaos Toolkit Extension for Microsoft Azure'
+version = '0.9.0'
```

Comment:
why this change?

Reply:
when I try to install chaostoolkit-azure as a standalone dependency without wheel packages or cache, 
like `pip install chaostoolkit-azure --no-cache-dir --no-binary :all:` import is failing with the following error:

```
Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-n1z_3n35/chaostoolkit-azure/setup.py", line 10, in <module>
        from chaosazure import __version__
      File "./chaosazure/__init__.py", line 7, in <module>
        from azure.mgmt.compute import ComputeManagementClient
    ModuleNotFoundError: No module named 'azure'
```

 importing `__version__` in setup.py causes this failure, because by importing this variable, the import statements in chaosazure/__init__.py are executed before requirements are resolved down below in the setup.py

So I think the version has to be hardcoded in the setup.py

Reply:
I see. Another way would be a utility function such as:

https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/master/setup.py#L10

then

https://github.com/chaostoolkit/chaostoolkit-kubernetes/blob/master/setup.py#L64